src/calcurse.c

```diff
@@ -143,7 +143,7 @@ static inline void key_add_item(void)
 
 static inline void key_edit_item(void)
 {
-	if (wins_slctd() == APP) {
+	if (wins_slctd() == APP && !event_dummy()) {
```

Comment:
This does not look right. And it does not work. This commits allows you to edit the description of a dummy item (which makes calcurse crash). You probably meant `ui_day_dummy()` here?

Reply:
See below.